### DISCUSSION TITLE
Meteor photo menu

### DISCUSSION THREAD

**Discussion by quanticleap** [aaronwmorris/indi-allsky#1508] [Q&A]
Hi, were I can find the Meteor photos, I looked in Media:Gallery, Images,Timelapses,minitimelapse and panorama?
I can see startrails,keogram and minitimelapses, but no Meteor filter.
I read the documentations and explain how to enable meteor detection and create mask, but I cannot find how the guide to see the meteor photos.Also I saw photos in "discussions" page, but the menu Meteor is not visible.

Thanks

**Comment by aaronwmorris** [comment]
On the Image Viewer page, there is a "Detections" switch you may activate to only show images with meteor/line detections.

**Comment by quanticleap** [comment]
Thank you Aaron!

Il lun 16 set 2024, 03:18 Aaron W Morris ***@***.***> ha
scritto:
